tools/buildtools/meson.build

```diff
@@ -0,0 +1,18 @@
+c_args = ['-pedantic', '-std=c99']
```

Comment:
-pedantic is handled by warning_level=3, the other one is c_std=c99. Note if this only applies to a handful of targets you can use it with `override_options: [...]` instead of project-wide default_options.